src/emulator.js

```diff
@@ -15,6 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import * as fcl from "@onflow/fcl";
```

Comment:
```suggestion
import { send, build, getBlock, decode } from "@onflow/fcl";
```

Reply:
We can import only methods we need. Then transpiler can `tree-shake` what is not necessary (we are not using all of the methods from `@onflow/fcl`) :)